posts/views.py

```diff
@@ -15,7 +15,7 @@ def post(self, request):
             title = request.data['title']
             content = request.data['content']
             author = request.author
```

Comment:
일단 여기가 문제인데, request 객체에는 author가 없으니 당연히 없는곳에 접근하려하면 에러가 나는게 맞습니다.
```suggestion
            author = request.data['author']
```